test/integration/narrow/routes/endemics/biosecurity.test.js

```diff
@@ -74,7 +74,7 @@ describe('Biosecurity test when Optional PI Hunt is OFF', () => {
         auth
       }
 
-      getEndemicsClaim.mockReturnValue({ typeOfLivestock: 'beef', reviewTestResults: 'negative' })
+      getEndemicsClaim.mockReturnValue({ typeOfLivestock: 'beef', reviewTestResults: 'negative', reference: '12345' })
```

Comment:
might be nice to make the references look a little more 'real' so it's obvious at a glance what we are dealing with.